services/media/service/.env.template

```diff
@@ -13,6 +13,10 @@ DATABASE_LOGIN=media_service_login
 DATABASE_LOGIN_PASSWORD=media_service_login_pwd
 DATABASE_GQL_ROLE=media_service_gql_role
 
+# NOTE: If these values change the old database replication slots must be explicitly dropped.
```

Comment:
In the `feature/localization` branch - we explicitly left slot env var value empty in the
.env.template and incorporated a few script changes to:
- make db:reset generate explicit unique slot names
- added a new script to be able to generate unique values for deployment.

If the current state is left and 2 different partners would deploy their
versions of media-template - second one would fail, because default replication
slot names would be used.

`feature/localization` branch also has a huge comment here describing this
stuff. Perhaps something similar can be made here.